app/templates/flatpages/privacy-policy.html

```diff
@@ -0,0 +1,18 @@
+{% extends "base.html" %}
+
+{% block head %}
+<title>Privacy Policy | HackItForward</title>
+{% endblock %}
+
+{% block body %}
+<space class="large"></space>
+<div class="content" style="margin-bottom: 90px;">
+  <h1 style="text-align: center; margin-bottom: 40px;">Privacy Policy</h1>
+  <ul>
+    <li>HackItForward stores a small amount of data such as your username, your profile picture and other user-generated data on the platform when you create an account.</li>
+    <li>Google and Github OAuth are used to facilitate the registration process.</li>
+    <li>All data is stored securely on a cloud server hosted with Digital Ocean. It will not be shared with any third party. </li>
```

Comment:
This.... doesn't really mean anything. I suppose you could clear it up a little saying something along the lines of "we take adequate precautions from external access, etc. with your data."

Reply:
Also, the third party comment isn't true for at *least* two reasons:
1. Above you just mentioned you can sign up with a third party
2. If we receive a legal request for data from the government.